components/SwipeRow.js

```diff
@@ -489,6 +551,30 @@ SwipeRow.propTypes = {
 	 * Callback invoked any time the swipe value of the row is changed
 	 */
 	onSwipeValueChange: PropTypes.func,
+	/**
+	 * TranslateX value for force-closing the row to the Left End (positive number)
+	 */
+	forceCloseToLeftThreshold: PropTypes.number,
+	/**
+	 * TranslateX value for force-closing the row to the Right End (negative number)
+	 */
+	forceCloseToRightThreshold: PropTypes.number,
+	/**
+	 * Callback invoked when row is force closing to the Left End
+	 */
+	onForceCloseToLeft: PropTypes.func,
+	/**
+	 * Callback invoked when row is force closing to the Right End
+	 */
+	onForceCloseToRight: PropTypes.func,
+	/**
+	 * Callback invoked when row has finished force closing to the Left End
+	 */
+	onForceCloseToLeftEnd: PropTypes.func,
+	/**
+	 * Callback invoked when row has finished force closing to the Right End
+	 */
+	onForceCloseToRightEnd: PropTypes.func
```

Comment:
👏 maybe add these to the docs as well? :)

Reply:
@jemise111 Yup! 👍

Reply:
@jemise111 Done! https://github.com/jemise111/react-native-swipe-list-view/pull/299/commits/1970a313a1a9768f04caff79e96a4461d899e248